src/main/java/com/hollingsworth/arsnouveau/common/entity/AmethystGolem.java

```diff
@@ -61,6 +63,7 @@ public class AmethystGolem extends PathfinderMob implements GeoEntity, IDispella
     public static final EntityDataAccessor<Boolean> IMBUEING = SynchedEntityData.defineId(AmethystGolem.class, EntityDataSerializers.BOOLEAN);
     public static final EntityDataAccessor<Boolean> STOMPING = SynchedEntityData.defineId(AmethystGolem.class, EntityDataSerializers.BOOLEAN);
     public static final EntityDataAccessor<BlockPos> IMBUE_POS = SynchedEntityData.defineId(AmethystGolem.class, EntityDataSerializers.BLOCK_POS);
+    public final List<BuddingConversionRecipe> recipes;
```

Comment:
this should probably be generated and cached with the datapack events, like the tomes are

Reply:
Good idea, will update

Reply:
Resolved in ad9e2547a30aa5fe3456e74bd3ea5a3a543d1685